app/views/error_template.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(layout: layout)
+@(pageTitle: String, heading: String, message: String)(implicit request: Request[_], messages: Messages, lang: Lang)
+
+@contentHeader = {
+    <h1 class="govuk-heading-xl">@heading</h1>
+}
+
+@mainContent = {
+    <p class="govuk-body">@message</p>
+}
+
+@content = {
+@contentHeader
+
+@mainContent
+}
+
+@scriptElement = {
+    <script src='@routes.Assets.versioned("jquery/jquery-1.11.3.min.js")'></script>
```

Comment:
Does this not need the same jquery versions as used above?
https://github.com/hmrc/ated-subscription-frontend/pull/133/files#diff-d070f29163ec0b2b667c213657c9a8efdae861a3fdbc8245c493066e1aaa12c9R30